bindings/src/main/resources/META-INF/services/io.pravega.segmentstore.storage.StorageFactoryFactory

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+io.pravega.storage.filesystem.FileSystemStorageFactoryFactory
```

Comment:
What is this file? is it needed? If so, please add a comment to it describing how its' supposed to be used

Reply:
Added comment explaining the use.